server/memphis_handlers_schemas.go

```diff
@@ -79,13 +118,94 @@ func validateSchemaContent(schemaContent, schemaType string) error {
 	return nil
 }
 
+func generateSchemaDescriptor(schemaName string, schemaVersionNum int, schemaContent, schemaType string) (string, error) {
+	if len(schemaContent) == 0 {
+		panic("attempt to generate schema descriptor with empty schema")
+	}
+
+	switch schemaType {
```

Comment:
the descriptor creation is relevant for protobuf only so no need to switch case in here